### DISCUSSION TITLE
walletconnect how to custom qrcode

### DISCUSSION THREAD

**Discussion by Chi111** [WalletConnect/walletconnect-monorepo#2531] [Developer Support]
hi
this is my version @wagmi/core": "0.9.4

import { WalletConnectConnector } from '@wagmi/core/connectors/walletConnect';
new WalletConnectConnector({
        chains,
        options: {
          qrcode: true,
        },
});

**Comment by finessevanes** [comment]
Hey these are wagmi packages:
@wagmi/core/connectors/walletConnect
@wagmi/core

I would give their docs a look: https://wagmi.sh/examples/connect-wallet
